src/js/tabs/map.js

```diff
@@ -76,31 +76,30 @@ function initializeMap() {
 }
 
 function processMapEvents(e) {
-
     try {
-        switch(e.data.action) {
-
-        case 'zoom_in':
-            mapView.setZoom(mapView.getZoom() + 1);
-            break;
-
-        case 'zoom_out':
-            mapView.setZoom(mapView.getZoom() - 1);
-            break;
-
-        case 'center':
-            iconFeature.setStyle(iconStyle);
-            const center = ol.proj.fromLonLat([e.data.lon, e.data.lat]);
-            mapView.setCenter(center);
-            iconGeometry.setCoordinates(center);
-            break;
-
-        case 'nofix':
-            iconFeature.setStyle(iconStyleNoFix);
-            break;
+        switch (e.data.action) {
+            case 'zoom_in':
+                mapView.setZoom(mapView.getZoom() + 1);
+                break;
+
+            case 'zoom_out':
+                mapView.setZoom(mapView.getZoom() - 1);
+                break;
+
+            case 'center':
+                iconFeature.setStyle(iconStyle);
+                const center = ol.proj.fromLonLat([e.data.lon, e.data.lat]);
+                mapView.setCenter(center);
+                const heading = e.data.heading === undefined ? 0 : e.data.heading;
+                mapView.setRotation(heading);
+                iconGeometry.setCoordinates(center);
+                break;
+
+            case 'nofix':
+                iconFeature.setStyle(iconStyleNoFix);
+                break;
         }
-
-  } catch (err) {
-      console.log(`Map error ${err}`);
-  }
+    } catch (err) {
+        console.error('Map error', err);
```

Comment:
Why remove string interpolation?

Reply:
So that we let the browser format the error object as it wishes and not force it to be a string.

Reply:
Plus, it won't really show since the map is running in its own webview with its own console. You have to have the console of the inner webview open to see the message anyway. It's bad, but I didn't take the time to look deeper in the issue